control/plugin/collector_proxy.go

```diff
@@ -28,22 +28,22 @@ import (
 
 // Arguments passed to CollectMetrics() for a Collector implementation
 type CollectMetricsArgs struct {
```

Comment:
It seems that struct type is not needed here but a type definition. Maybe there will be more fields be added later?